WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanThreatDetailsCoordinator.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+protocol JetpackScanThreatDetailsView {
+    func showFixThreatSuccess()
+    func showIgnoreThreatSuccess()
+    func showFixThreatError()
+    func showIgnoreThreatError()
+}
+
+class JetpackScanThreatDetailsCoordinator {
+
+    // MARK: - Properties
+
+    private let service: JetpackScanService
+    private let blog: Blog
+    private let threat: JetpackScanThreat
+    private let view: JetpackScanThreatDetailsView
+
+    // MARK: - Init
+
+    init(blog: Blog,
+         threat: JetpackScanThreat,
+         view: JetpackScanThreatDetailsView,
+         service: JetpackScanService? = nil,
+         context: NSManagedObjectContext = ContextManager.sharedInstance().mainContext) {
+
+        self.service = service ?? JetpackScanService(managedObjectContext: context)
+        self.blog = blog
+        self.threat = threat
+        self.view = view
+    }
+
+    // MARK: - Public
+
+    public func fixThreat() {
```

Comment:
Hmm, I'm wondering if we could use a delegate attached to the `JetpackScanViewController` that triggers the fix / ignore methods since that view does a lot of heavy lifting for the fixing / ignoring. 


Thoughts?

Reply:
Ahh, that's why there was an `ignoreThreat` method already defined in `JetpackScanCoordinator`!
Fixed! 2462d66